runelite-client/src/main/java/net/runelite/client/plugins/chatchannel/ChatChannelPlugin.java

```diff
@@ -845,4 +855,17 @@ private void colorIgnoredPlayers(Color ignoreColor)
 			listWidget.setTextColor(ignoreColor.getRGB());
 		}
 	}
+
+	private void showClanOnlineMemberCount (boolean show)
+	{
+		Widget clanTitle = client.getWidget(WidgetInfo.CLAN_TITLE);
+		ClanChannel clanChannel = client.getClanChannel();
```

Comment:
It would be good to show the guest channel count too probably.

Reply:
Added a separate option for guest clan chat.